collector/lib/CollectorConfig.cpp

```diff
@@ -47,7 +47,7 @@ BoolEnvVar set_import_users("ROX_COLLECTOR_SET_IMPORT_USERS", false);
 
 BoolEnvVar collect_connection_status("ROX_COLLECT_CONNECTION_STATUS", true);
 
-BoolEnvVar enable_external_ips("ROX_ENABLE_EXTERNAL_IPS", false);
+BoolEnvVar enable_external_ips("ROX_EXTERNAL_IPS", false);
```

Comment:
I actually think the `ENABLE` makes it clearer what the env variable is for. But I would argue a variable in the form `<PROGRAM>_<FEATURE>_<PARAM>` is even more clear, so in this case we might want to name the variable `ROX_EXTERNAL_IPS_ENABLE` or `ROX_COLLECTOR_EXTERNAL_IPS_ENABLE`

Reply:
In general I think I agree. @ovalenti what do you think? should we update the central feature flag?

Reply:
In Central, it looks like the convention is to use short names and no verb... https://github.com/stackrox/stackrox/blob/0b83c7607cb58d28e0165e20936f30d0c11d12c0/pkg/features/list.go

Although unifying the flag names seemed like a good idea in the first place, the collector one is likely not going to be used directly by any customer. So I am starting to think that we could keep the names as they were, or as Mauro suggests. If it was only my, the thing would not be named `EXTERNAL_IP` at all...

Reply:
I don't think `ENABLE` is needed in the name. In stackrox/stackrox the function to check the value of feature flags is `Enable()`, so it would seem to be redundant.

Reply:
> In stackrox/stackrox the function to check the value of feature flags is `Enable()`, so it would seem to be redundant.

It wouldn't be redundant to a user that is trying to configure collector without any knowledge of what the code in the main repo looks like 🙂

Reply:
I will revert this for now and we can decide on naming conventions later on